regimes/au/tax_categories.go

```diff
@@ -0,0 +1,128 @@
+package au
+
+import (
+	"time"
+
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/tax"
+)
+
+var taxCategories = []*tax.CategoryDef{
+	{
+		Code: tax.CategoryGST,
+		Name: i18n.String{
+			i18n.EN: "GST",
+		},
+		Title: i18n.String{
+			i18n.EN: "Goods and Services Tax",
+		},
+		Sources: []*tax.Source{
+			{
+				Title: i18n.String{
+					i18n.EN: "How GST Works",
+				},
+				URL: "https://www.ato.gov.au/businesses-and-organisations/gst-excise-and-indirect-taxes/gst/how-gst-works",
```

Comment:
I couldn't find the definitions of the actual tax rates here... the objective here is to have a specific page that can be used to determine what the latest rates are.